monitoring/ceph-mixin/grafana-json-to-libsonnet.py

```diff
@@ -0,0 +1,23 @@
+import json, sys
+from jinja2 import Environment, FileSystemLoader
+
+if (len(sys.argv[1:]) < 2):
+    raise Exception(
+        "Need to pass two parameters, like: python3 grafana-libsonnet-to-json.py <input_json_path> <output_libsonnet_path>"
+    )
+
+json_path = sys.argv[1] # example: dashboards_out/new.json
+output_path = sys.argv[2] # example: dashboards/new.libsonnet
+
+with open(json_path, "r") as f:
+    data = json.load(f)
```

Comment:
I'd suggest to use a bit more meaningful word as the variable name rather than `f` or `data`. It'd make it more easy to maintain this long term. The same goes for others too.